java-core/google-cloud-core-http/src/main/java/com/google/cloud/http/HttpTransportOptions.java

```diff
@@ -42,6 +42,7 @@
 public class HttpTransportOptions implements TransportOptions {
 
   private static final long serialVersionUID = 7890117765045419810L;
+  private static final String GOOGLE_CLOUD_UNIVERSE_DOMAIN = "GOOGLE_CLOUD_UNIVERSE_DOMAIN";
```

Comment:
Can we expose the same [String](https://github.com/googleapis/sdk-platform-java/blob/main/gax-java/gax/src/main/java/com/google/api/gax/rpc/EndpointContext.java#L50) as public in Gax?

Reply:
Ah, yep. I think so. I'll try it.